shanoir-ng-studies/src/main/resources/application.yml

```diff
@@ -131,6 +131,8 @@ ms.url:
   shanoir-ng-studycards: http://localhost:9904/studycards
   shanoir-ng-examinations: http://localhost:9904/examinations
   shanoir-ng-bids: http://localhost:9904/bids
+# data paths
+studies-data: /tmp
```

Comment:
I don't think this should be here

Reply:
it is in dev profile only, and to better test locally it is fine?

Reply:
Ok, why not